lib/get_order_command.rb

```diff
@@ -0,0 +1,21 @@
+require 'order'
+
+class GetOrderCommand
+  attr_reader :response
+
+  def initialize(message)
+    @user_message = message
+  end
+
+  def response?
+    response
+  end
+
+  def run()
+    user_message = @user_message.gsub("order: ", "")
+    user_id_meal_researched = user_message[/(?<=\<@)(\w+)(?=>)/]
```

Comment:
It's not very clear to me what `user_id_meal_researched` means. Is that the user id of the user who asked for their meal, or is it the meal that the user chose before?

Using a positive lookbehind in the regular expression seem overkill to me here.
You can simplify the expression itself, too, and then use Ruby's `match` to extract the data out of the string.
```
/<@([^>]+)>/.match(user_message) do |match|
...
end
```